components/com_content/tmpl/category/blog.php

```diff
@@ -66,9 +66,10 @@
 	<?php endif; ?>
 
 	<?php if (empty($this->lead_items) && empty($this->link_items) && empty($this->intro_items)) : ?>
-		<?php if ($this->params->get('show_no_articles', 1)) : ?>
-			<p><?php echo Text::_('COM_CONTENT_NO_ARTICLES'); ?></p>
-		<?php endif; ?>
+		<div class="alert alert-info">
+			<span class="icon-info-circle" aria-hidden="true"></span><span class="visually-hidden"><?php echo Text::_('INFO'); ?></span>
+				<?php echo Text::_('COM_CONTENT_NO_ARTICLES'); ?>
+		</div>
```

Comment:
```suggestion
		<?php if ($this->params->get('show_no_articles', 1)) : ?>
			<div class="alert alert-info">
				<span class="icon-info-circle" aria-hidden="true"></span><span class="visually-hidden"><?php echo Text::_('INFO'); ?></span>
					<?php echo Text::_('COM_CONTENT_NO_ARTICLES'); ?>
			</div>
		<?php endif; ?>
```
Can it be that the if has been removed by accident?

Comparing with the change in the other file I think it should look as suggested here.

Reply:
thanks